ardent/distribution.yaml

```diff
@@ -114,2024 +99,739 @@ repositories:
-      version: master
-    status: maintained
-  rosidl_typesupport_c:
-    release:
-      tags:
-        release: release/ardent/{package}/{version}
-      type: git
-      url: https://github.com/ros2-gbp/rosidl_typesupport-release.git
-      version: 0.0.3-1
-    source:
-      type: git
-      url: https://github.com/ros2/rosidl_typesupport.git
-      version: master
-    status: maintained
-  rosidl_typesupport_cpp:
-    release:
-      tags:
-        release: release/ardent/{package}/{version}
-      type: git
-      url: https://github.com/ros2-gbp/rosidl_typesupport-release.git
-      version: 0.0.3-1
-    source:
-      type: git
-      url: https://github.com/ros2/rosidl_typesupport.git
-      version: master
-    status: maintained
-  rosidl_typesupport_interface:
-    release:
-      tags:
-        release: release/ardent/{package}/{version}
-      type: git
-      url: https://github.com/ros2-gbp/rosidl-release.git
-      version: 0.0.3-1
-    source:
-      type: git
-      url: https://github.com/ros2/rosidl.git
-      version: master
-    status: maintained
-  rosidl_typesupport_introspection_c:
-    release:
-      tags:
-        release: release/ardent/{package}/{version}
-      type: git
-      url: https://github.com/ros2-gbp/rosidl-release.git
-      version: 0.0.3-1
-    source:
-      type: git
-      url: https://github.com/ros2/rosidl.git
-      version: master
-    status: maintained
-  rosidl_typesupport_introspection_cpp:
-    release:
-      tags:
-        release: release/ardent/{package}/{version}
-      type: git
-      url: https://github.com/ros2-gbp/rosidl-release.git
-      version: 0.0.3-1
-    source:
-      type: git
-      url: https://github.com/ros2/rosidl.git
-      version: master
-    status: maintained
-  rosidl_typesupport_opensplice_c:
-    release:
-      tags:
-        release: release/ardent/{package}/{version}
-      type: git
-      url: https://github.com/ros2-gbp/rmw_opensplice-release.git
-      version: 0.0.3-2
-    source:
-      type: git
-      url: https://github.com/ros2/rmw_opensplice.git
-      version: master
-    status: maintained
-  rosidl_typesupport_opensplice_cpp:
-    release:
-      tags:
-        release: release/ardent/{package}/{version}
-      type: git
-      url: https://github.com/ros2-gbp/rmw_opensplice-release.git
-      version: 0.0.3-2
+      url: https://github.com/ros2-gbp/googletest-release.git
+      version: 1.8.0-3
     source:
       type: git
-      url: https://github.com/ros2/rmw_opensplice.git
-      version: master
+      url: https://github.com/ament/googletest.git
+      version: ros2
     status: maintained
-  rttest:
+  joystick_drivers:
     release:
+      packages:
+      - joy
       tags:
         release: release/ardent/{package}/{version}
       type: git
-      url: https://github.com/ros2-gbp/realtime_support-release.git
-      version: 0.0.3-1
+      url: https://github.com/ros2-gbp/joy-release.git
```

Comment:
TODO(@nuclearsandwich) fix the name of this release repo `joy-release` -> `joystick_drivers-release`